### PR TITLE
Create function for Boolean HoT columns

### PR DESCRIPTION
**Pull request by apmasell**
JIRA: GLT-1419

### CODE REVIEW

File: miso-web/src/main/webapp/scripts/hot.js [hunk 1]
```diff
@@ -496,6 +496,28 @@ var HotUtils = {
     };
     return baseobj;
   },
+  makeColumnForBooleanMaybe : function(headerName, include, property) {
+    return {
+      header : headerName,
+      data : property,
+      type : 'dropdown',
+      trimDropdown : false,
+      source : [ 'unknown', 'true', 'false' ],
```
Comment (callunity) [inline miso-web/src/main/webapp/scripts/hot.js]: Title case, please.

Comment (apmasell) [inline miso-web/src/main/webapp/scripts/hot.js]: Fixed.

File: miso-web/src/main/webapp/scripts/hot.js [hunk 2]
```diff
@@ -496,6 +496,36 @@ var HotUtils = {
...
     return baseobj;
   },
+  makeColumnForBooleanMaybe : function(headerName, include, property) {
+    return {
+      header : headerName,
+      data : property,
+      type : 'dropdown',
+      trimDropdown : false,
+      source : [ 'Unknown', 'True', 'False' ],
+      include : true,
```
Comment (djcooke) [inline miso-web/src/main/webapp/scripts/hot.js]: Should depend on the `include` parameter

File: miso-web/src/main/webapp/scripts/hot.js [hunk 3]
```diff
@@ -496,6 +496,36 @@ var HotUtils = {
     };
     return baseobj;
   },
+  makeColumnForBooleanMaybe : function(headerName, include, property) {
```
Comment (djcooke) [inline miso-web/src/main/webapp/scripts/hot.js]: "OptionalBoolean" would be clearer than "BooleanMaybe," but :man_shrugging:

**Comment by apmasell** [comment]
Fixed.

**Pull request merged**